examples/czmq/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required (VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("${CMAKE_CURRENT_SOURCE_DIR}/../common.cmake")
+
+project(download-czmq)
+
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+
+hunter_add_package(ZeroMQ)
```

Comment:
No, you are right, this and the next comment are both remains of a previous code. I will remove the lines and squash everything